app/BUILD.bazel

```diff
@@ -336,6 +388,70 @@ kt_android_library(
     visibility = ["//visibility:private"],
 )
 
+# Library for all custom data-binding adapters under org/oppia/app/databinding
+android_library(
+    name = "databinding_adapters",
+    custom_package = "org.oppia.app.impl.adapters",
```

Comment:
The inconsistent custom packages is a bit confusing--can we use a consistent pattern for these?

Reply:
Fixed! I basically have the package name mirroring the library name - but I just replaced `_` with `.`  :)